src/mail/MailEditor.js

```diff
@@ -713,15 +713,21 @@ export class MailEditor {
 					label: "download_action",
 					type: ButtonType.Secondary,
 					click: () => {
+						let promise = Promise.resolve()
```

Comment:
don't need to assign here and I wouldn't to catch errors. Flow can prove that we assign it in all branches.